packages/api-react/package.json

```diff
@@ -9,11 +9,12 @@
   },
   "main": "dist/index.js",
   "module": "dist/esm.js",
+  "types": "./dist/types",
   "scripts": {
     "build": "npm run build:js",
-    "build:js": "rollup -c",
+    "build:js": "rollup -c && tsc --emitDeclarationOnly",
     "build:ci": "npm run build:js",
-    "build:watch": "rollup -c -w",
+    "build:watch": "tsc --emitDeclarationOnly --watch & rollup -c -w",
```

Comment:
Only `tsc --emitDeclarationOnly` part was working and never reached to `rollup`, even changing `&` to `&&`.
Maybe you need to use `concurrent` to run multiple watchers at once.

Reply:
https://user-images.githubusercontent.com/84098616/230444399-aa72a8b1-651b-4d21-b9b5-b6d6ce967cc8.mp4

This was my experience.
Only type declaration files were created but no transpiled js files there.

Reply:
Hi, I've pushed the commit which fixes Windows issue.